index.html

```diff
@@ -33,5 +42,141 @@ <h3>I'm a software Developer, </h3>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+          <button class="see-project btn-change"><a href="#">See project</a></button>
+        </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+    </section>
+    <section class="section-four">
+      <div class="about-me-ctn">
+        <h1>About me</h1>
+        <p>
+          Hello I&apos;m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don&apos;t hesitate to contact me.
+        </p>
+        <button class="btn-change"><a href="#">Get my resume</a></button>
+      </div>
+    </section>
```

Comment:
- [ ] Great work on adding button styles! However, there are still some adjustments needed:

- According to your previous reviewer The button styles for the active and hover states are currently missing. To ensure a consistent and visually appealing user experience, please add appropriate styles for these states as specified in the design.